apps/prod/kustomization.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../base/cert-manager
+- ../base/dendrite
+- ../base/cryptpad
+- ../base/external-dns
+- ../base/grafana
+- ../base/prometheus
```

Comment:
why are all of these paths prefixed with `base`? what makes an app belong into `base`?